testing/kokoro/system_tests.sh

```diff
@@ -100,6 +100,7 @@ export GCLOUD_ORGANIZATION=1081635000895
 export SCC_PUBSUB_PROJECT="project-a-id"
 export SCC_PUBSUB_TOPIC="projects/project-a-id/topics/notifications-sample-topic"
 export SCC_PUBSUB_SUBSCRIPTION="notification-sample-subscription"
+export SCC_BIGQUERY_DATASET_NAME="projects/project-a-id/datasets/dataset-id"
```

Comment:
issue: remove, since this is now hard-coded into the sample.

Reply:
Addressed.